src/Covid19Dashboard/ControlPanel/index.jsx

```diff
@@ -1,18 +1,48 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
+import Tooltip from 'rc-tooltip';
+import { ExclamationCircleOutlined } from '@ant-design/icons';
 import LegendPanel from './LegendPanel';
 import MapStylePanel from './MapStylePanel';
 import LayerSelector from './LayerSelector';
+import DataSelector from './DataSelector';
 
 import './ControlPanel.less';
 
 class ControlPanel extends PureComponent {
   render() {
+    const tooltipText1 = 'Google Mobility Data shows movement trends as a percentage variance from an established baseline for a given day and place.';
+    const tooltipText2 = 'Additional data of interest to display in map tool-tip pop-up.  These data will not influence color gradients on the map.';
+
     return (
       <div className='control-panel'>
-        <h3>Confirmed cases</h3>
+        <h3>{this.props.legendTitle}</h3>
         <p>
-          Data source: <a href='https://systems.jhu.edu'>Johns Hopkins University CSSE</a>
+          Data source:&nbsp;
+          <a href={this.props.legendDataSource.link}>
```

Comment:
Can you add ` target="_blank"` to this anchor tag so that the link opens in a new tab?

Reply:
added